### ISSUE TITLE
GUI believes a Coordinate is clamped when it is not

### ISSUE THREAD

**Issue by aseth1** [opensim-org/opensim-gui#780]
### Steps to reproduce
Load pendulum model. Set any of the unclamped coordinates to a value that exceed -90 to 90.

### Expected result
Change is accepted with no error and model changes pose since coordinates are not clamped.

### Actual result
Receive two error message pop-ups that "Value enetered is outside range of clamped coordinate. Ignoring"
![image](https://user-images.githubusercontent.com/6494122/40521591-2b4f17ee-5f81-11e8-9295-6fcbd6eb838c.png)

Clamping and unclamping enables the user to set the value without error.

Also recommend changing the message in the case it is clamped to the suggested message in #534.

### Environment and GUI version
Windows OpenSim-5eacd8a5-2018-05-22

**Comment by aseth1** [comment]
The main issue appears to fixed on Windows OpenSim_cea85df3-2018-07-18. No warning popup when editing an unclamped coordinate beyond its limits.

However, if you then clamp while exceeding the limits, nothing appears to happen but an `Exception` is thrown (⛔️ appears) with the following message:
![clampingerror](https://user-images.githubusercontent.com/6494122/43104818-84b8e406-8e87-11e8-8db9-9d0539ecfd04.PNG)

And coordinate sliders no longer have any effect on the model.

I think the dialog should appear again with a message that is relevant to both cases: 1. value entered is beyond clamped range, and 2. clamp is turned on when coordinate value is outside of range and value is updated to be nearest in range.

**Comment by aymanhab** [comment]
I agree regarding the end result being a dialog. The only reason this doesn't happen on its own is that the Exception thrown is a ow level SimTK::Exception rather than an OpenSim::Exception that the GUI knows how to handle (thus truly Unexpected Exception)  😃
